src/Symfony/Component/DependencyInjection/Argument/LazyClosure.php

```diff
@@ -25,26 +24,9 @@
  */
 class LazyClosure
 {
-    public readonly object $service;
```

Comment:
isn't it a BC break to change the property from public to protected ?

Reply:
`@internal` to the rescue ;)